benchcab/main.py

```diff
@@ -16,6 +17,13 @@ def parse_and_dispatch(parser):
         Parser object.
     """
     args = vars(parser.parse_args(sys.argv[1:] if sys.argv[1:] else ["-h"]))
+
+    # Intercept the verbosity flag to engage the logger
+    log_level = 'debug' if args.get('verbose', False) == True else 'info'
+
+    # We just need to instantiate this with the desired level
+    logger = get_logger(level=log_level)
```

Comment:
Considering the suggestion above this would then need to change to:

```suggestion
    # We just need to instantiate this with the desired level
    logger = get_logger(level=args.log_level)
```

Reply:
See comment above :)